lib/peer-info.js

```diff
@@ -78,4 +82,11 @@ module.exports = class PeerInfo {
   ban (val) {
     this.banned = !!val
   }
+  
+  topic (topic) {
```

Comment:
should be private no?

Reply:
I think it was like this in previous version, and I thought should not be private as we may need to access externally, e.g. in `.on('connection'..` (I may not properly understand public/private conventions used however).

Reply:
I see it's amended in later commit, can ignore my last comment